decidim-forms/spec/types/questionnaire_type_spec.rb

```diff
@@ -54,7 +54,7 @@ module Forms
 
       describe "forEntity" do
         let(:query) { "{ forEntity { id } }" }
-        let(:meeting) { create(:meeting) }
+        let(:meeting) { create(:meeting, :published) }
```

Comment:
Can we add an example for when the meeting is not published 🙏🏽 ?

Reply:
done in 0c35edfde8